etc/httpd/global.conf

```diff
@@ -32,24 +32,9 @@ RewriteRule ^(.*)$ $1 [L,R=301]
 # bug 845988 - remove double slashes in URLs
 RewriteRule ^(.*)//(.*)$ $1/$2 [L,R=301]
 
-# bug 764261, 841393, 996608, 1008162, 1067691, 1113136, 1119022, 1131680
+# bug 764261, 841393, 996608, 1008162, 1067691, 1113136, 1119022, 1131680, 1115626
 RewriteRule ^/zh-TW/$ http://mozilla.com.tw/ [L,R=301]
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox/dnt
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox/geolocation
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox/independent
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox/os/devices
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox/tiles
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox/android
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox/sync
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox/partners
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox/desktop/tips
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox/nightly/firstrun/?
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox/hello
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox(/(?:\d+\.\d+\.?(?:\d+)?\.?(?:\d+)?(?:[a|b]?)(?:\d*)(?:pre)?(?:\d)?))?/hello/start/?
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox(/(?:\d+\.\d+\.?(?:\d+)?\.?(?:\d+)?(?:[a|b]?)(?:\d*)(?:pre)?(?:\d)?))?/firstrun/?
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox(/(?:\d+\.\d+\.?(?:\d+)?\.?(?:\d+)?(?:[a|b]?)(?:\d*)(?:pre)?(?:\d)?))?/tour/?
-RewriteCond %{REQUEST_URI} !^/zh-TW/firefox(/(?:\d+\.\d+\.?(?:\d+)?\.?(?:\d+)?(?:[a|b]?)(?:\d*)(?:pre)?(?:\d)?))?/whatsnew/?
-RewriteRule ^/zh-TW/firefox(/.*)?$ http://mozilla.com.tw/firefox$1 [L,R=301]
+RewriteCond %{REQUEST_URI} !^/zh-TW/firefox
 RewriteRule ^/zh-TW/mobile/?$ http://mozilla.com.tw/firefox/mobile/ [L,R=301]
 RewriteRule ^/zh-TW/download/?$ http://mozilla.com.tw/firefox/download/ [L,R=301]
 RewriteRule ^/zh-TW/products(/.*)?$ http://mozilla.com.tw/products$1 [L,R=301]
```

Comment:
The general /products/ page will soon be redirected to /firefox/products/ (see bug 1148127). No need to change the rule in this PR but I'll be changing it in another PR in the next few days.